crates/rolldown/tests/esbuild/default/dynamic_import_with_expression_cjs/artifacts.snap

```diff
@@ -1,5 +1,6 @@
 ---
 source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
```

Comment:
Try delete all changes and do a update again. This should remove unchanged snapshots.

Reply:
This pr is not finished yet, I will clean up in final step.